WordPressAuthenticator/Unified Auth/View Related/Password/PasswordViewController.swift

```diff
@@ -338,8 +344,9 @@ private extension PasswordViewController {
     func loadRows() {
         rows = [.gravatarEmail]
 
-        // Instructions only for social accounts
-        if loginFields.meta.socialService != nil {
+        // Instructions only for social accounts and simplified WPCom login flow
+        if loginFields.meta.socialService != nil ||
+            (configurations.enableSimplifiedLoginI1 && isSiteCredentialPassword == false) {
```

Comment:
nit: similar to before - instead of a general `enableSimplifiedLoginI1` that is used in various screens here and there, how about moving the `WordPressAuthenticatorDisplayStrings.wpcomPasswordInstructions` to `WordPressAuthenticatorConfiguration.wpcomPasswordInstructions: String?` with an optional value that's shown only when it's non-nil?